Microsoft.HealthVault.Fhir/Codes/HealthVault/HealthVaultVitalStatisticsCodes.cs

```diff
@@ -15,14 +16,14 @@ namespace Microsoft.HealthVault.Fhir.Codes.HealthVault
     /// </summary>
     public static class HealthVaultVitalStatisticsCodes
     {
-        public const string System = "http://healthvault.com/vocabularies/vital-statistics";
+        public static readonly string System = VocabularyUris.HealthVaultVocabulariesUri;
 
-        internal static Coding BodyWeight = new Coding()
+        internal static readonly Coding s_bodyWeight = new Coding()
```

Comment:
I am going to turn it public, it makes more sense that way actually.

---
In reply to: [125143733](https://github.com/Microsoft/healthvault-fhir-library/pull/3#discussion_r125143733) [](ancestors = 125143733,125125550)